docs/File-Formats.md

```diff
@@ -549,39 +547,38 @@ profile_name: Mutations
 data_filename: brca_tcga_pub.maf
 ```
 
-#### Data file
-A minimal mutation annotations file can contain just three of the MAF columns plus one annotation column, which is normally added to the end of each MAF row:
-
-* **Hugo_Symbol**: (MAF column) A [HUGO](http://www.genenames.org/) gene symbol.
-* **Tumor_Sample_Barcode**: (MAF column) This is the sample ID as listed in the clinical data file.
-* **Variant_Classification**: (MAF column) Translational effect of variant allele. Allowed values (from [Mutation Annotation Format page](https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification)): `Frame_Shift_Del, Frame_Shift_Ins, In_Frame_Del, In_Frame_Ins, Missense_Mutation, Nonsense_Mutation, Silent, Splice_Site, Translation_Start_Site, Nonstop_Mutation, 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR, Intron, RNA, Targeted_Region, De_novo_Start_InFrame, De_novo_Start_OutOfFrame`. cBioPortal skips the following types during the import: `Silent, Intron, 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR and RNA`. Two extra values are allowed by cBioPortal here as well: `Splice_Region, Unknown`. :warning: the values should be in the correct case. E.g. `missense_mutation` is not allowed, while `Missense_Mutation` is.
-* **HGVSp_Short**: (annotation column) Amino Acid Change, e.g. p.V600E.
+### Data file
+The mutation data file extends the [Mutation Annotation Format](https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification) (MAF) created as part of the [Cancer Genome Atlas](https://wiki.nci.nih.gov/display/TCGA/TCGA+Home) project, by adding *extra annotations* to each mutation record. This section describes two types of MAF files:
```

Comment:
Link to The Cancer Genome Atlas should include the 'The'

Reply:
Done